app/views/devise/registrations/edit.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="col-sm-4">
     <h1><%= t('.title', resource: resource_name.to_s.humanize) %></h1>
 
-    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: {method: :put}) do |f| %>
```

Comment:
Let's make sure we have a rule for that in `rubocop` and it's checked in `.erb` files, start with https://rubocop.readthedocs.io/en/latest/cops_layout/#layoutspaceinsidehashliteralbraces